nodeup/pkg/model/containerd.go

```diff
@@ -407,12 +407,17 @@ iptables -w -t nat -A IP-MASQ -d {{.NonMasqueradeCIDR}} -m comment --comment "ip
 iptables -w -t nat -A IP-MASQ -m comment --comment "ip-masq: outbound traffic is subject to MASQUERADE (must be last in chain)" -j MASQUERADE
 `
 
-	if b.Cluster.Spec.Networking.NonMasqueradeCIDR == "" {
-		// We could fall back to the pod CIDR, that is likely more correct anyway
-		return fmt.Errorf("NonMasqueradeCIDR is not set")
+	nonMasqueradeCIDR := b.Cluster.Spec.Networking.NonMasqueradeCIDR
+	if nonMasqueradeCIDR == "" {
+		if podCIDR := b.Cluster.Spec.Networking.PodCIDR; podCIDR != "" {
+			klog.Warningf("defaulting nonMasqueradeCIDR to podCIDR %q", podCIDR)
+			nonMasqueradeCIDR = podCIDR
```

Comment:
It's going to masquerade the `ServiceClusterIPRange`? That doesn't seem correct.

Reply:
The service ips are dealt with "earlier" ... if we put a service IP onto the wire (i.e. dev eth0), then - yes - we would NAT it.